substrate/frame/metadata-hash-extension/src/lib.rs

```diff
@@ -68,12 +69,16 @@ enum MetadataHash {
 	Custom([u8; 32]),
 }
 
+static RUNTIME_METADATA: Option<[u8; 32]> = match option_env!("RUNTIME_METADATA_HASH") {
```

Comment:
```suggestion
const RUNTIME_METADATA: Option<[u8; 32]> = match option_env!("RUNTIME_METADATA_HASH") {
```

Can we not make this into a const?